platform/platform-app/pom.xml

```diff
@@ -395,75 +440,83 @@
+        <cxf.pax.exam.version>4.10.0</cxf.pax.exam.version>
         <cxf.felix.framework.version>2.0.5</cxf.felix.framework.version>
         <cxf.felix.configadmin.version>1.2.4</cxf.felix.configadmin.version>
         <cxf.xmlbeans.bundle.version>2.6.0_2</cxf.xmlbeans.bundle.version>
+        <cxf.servicemix.aspectj.version>1.8.7_2</cxf.servicemix.aspectj.version>
+        <cxf.servicemix.cglib.version>3.2.0_1</cxf.servicemix.cglib.version>
+        <cxf.servicemix.jodatime.version>2.3_1</cxf.servicemix.jodatime.version>
+        <cxf.servicemix.jsoup.version>1.5.2_2</cxf.servicemix.jsoup.version>
         <cxf.servicemix.specs.version>2.6.0</cxf.servicemix.specs.version>
         <cxf.servicemix.specs.version.jaxrs>2.6.0</cxf.servicemix.specs.version.jaxrs>
         <cxf.javax.mail.version>1.4.4</cxf.javax.mail.version>
         <cxf.jaxb.bundle.version>2.2.11_1</cxf.jaxb.bundle.version>
         <cxf.saaj-impl.bundle.version>1.3.23_2</cxf.saaj-impl.bundle.version>
         <cxf.stax-ex.version>1.7.6</cxf.stax-ex.version>
         <cxf.wsdl4j.bundle.version>1.6.3_1</cxf.wsdl4j.bundle.version>
-        <cxf.xmlsec.bundle.version>${xmlsec.version}</cxf.xmlsec.bundle.version>
+        <cxf.xmlsec.bundle.version>2.0.8</cxf.xmlsec.bundle.version>
         <cxf.rhino.bundle.version>1.7R2_3</cxf.rhino.bundle.version>
         <cxf.xmlresolver.bundle.version>1.2_5</cxf.xmlresolver.bundle.version>
         <cxf.xerces.bundle.version>2.11.0_1</cxf.xerces.bundle.version>
         <cxf.bcel.bundle.version>5.2_4</cxf.bcel.bundle.version>
+        <cxf.xalan.version>2.7.2</cxf.xalan.version>
         <cxf.xalan.bundle.version>2.7.2_3</cxf.xalan.bundle.version>
         <cxf.oro.bundle.version>2.0.8_6</cxf.oro.bundle.version>
         <cxf.dom4j.bundle.version>1.6.1_5</cxf.dom4j.bundle.version>
         <cxf.jdom.bundle.version>1.1_4</cxf.jdom.bundle.version>
         <cxf.olingo.version>1.2.0</cxf.olingo.version>
-        <cxf.tika.version>1.7</cxf.tika.version>
+        <cxf.tika.version>1.14</cxf.tika.version>
         <cxf.jexl.version>2.1.1</cxf.jexl.version>
         <cxf.htrace.version>4.0.1-incubating</cxf.htrace.version>
-        <cxf.checkstyle.extension/>
-        <cxf.jaxb.context.class/>
+        <cxf.checkstyle.extension />
+        <cxf.jaxb.context.class />
         <cxf.jaxb.context.class.property>none</cxf.jaxb.context.class.property>
         <cxf.spring.validation.mode>VALIDATION_AUTO</cxf.spring.validation.mode>
         <cxf.downloadSources>true</cxf.downloadSources>
-        <cxf.pmd.eclipse.ruleset.ext/>
-        <cxf.resources.base.path/>
-        <!-- End apache-cxf-3.1.4-src/parent/pom.xml-->
+        <cxf.pmd.eclipse.ruleset.ext />
+        <cxf.resources.base.path />
+        <!-- End apache-cxf-3.1.11-src/parent/pom.xml-->
         <!-- Copied from apache-cxf-3.0.4-src/pom.xml -->
-        <cxf.build-utils.version>3.1.0</cxf.build-utils.version>
         <cxf.xjc-utils.version>3.0.5</cxf.xjc-utils.version>
```

Comment:
Good catch. This is actually copied from https://github.com/apache/cxf/blob/cxf-3.1.11/pom.xml#L44. I'll update the comment.